### ISSUE TITLE
Community fork?

### ISSUE THREAD

**Issue by daniel-sc** [podio/podio-java#39]
Hello,

would any one be interested in maintaining this client as a community projekt (analogous to the php client - see: https://github.com/podio-community/podio-php/issues/153)?

Potentially interested people:
@froilan @thepratt @RSpace @pulczynski @mhrdev @filchambers 

Best, Daniel

**Comment by thepratt** [comment]
No, sorry @daniel-sc. I no longer work at a company where Podio is used. We worked around this in the past by writing our own Scala client wrapping the requests. I don't think it ever got to a point that it could be open-sourced; Podio was potentially deprecated in favour of a more relevant in-house system.

**Comment by daniel-sc** [comment]
I created https://github.com/podio-community/podio-java
